setup.py

```diff
@@ -60,7 +60,7 @@
     include_package_data=True,
     install_requires=[
         "importlib-metadata ; python_version<'3.8'",
-        "eth-ape>=0.1.0a20,<0.2.0",
+        "eth-ape>=0.1.0b1,<0.2.0",
```

Comment:
We should add `eth-account` and `eth-typing` here with whatever versions you have

Reply:
Might as well make it `0b2`